src/renderer/components/layout/sidebar-item.tsx

```diff
@@ -0,0 +1,90 @@
+import "./sidebar-item.scss";
+
+import React from "react";
+import { cssNames, prevDefault } from "../../utils";
+import { observer } from "mobx-react";
+import { NavLink } from "react-router-dom";
+import { Icon } from "../icon";
+import { TabLayoutRoute } from "./tab-layout";
+import { sidebarStorage } from "./sidebar-storage";
+
+interface SidebarItemProps {
+  id: string; // Used to save nav item collapse/expand state in local storage
+  url: string;
+  text: React.ReactNode | string;
+  className?: string;
+  icon?: React.ReactNode;
+  isHidden?: boolean;
+  isActive?: boolean;
+  subMenus?: TabLayoutRoute[];
+  onToggle?(id: string, meta: { props: SidebarItemProps, event: React.MouseEvent }): void;
+}
+
+@observer
+export class SidebarItem extends React.Component<SidebarItemProps> {
+  static displayName = "SidebarItem";
```

Comment:
Probably we should start using this static field in other places too to improve debugging react-components in `production`.

Reply:
Should that be a PR of its own?

Reply:
Maybe, it's just an example..